test/TestProjects/Spread-Cadl/Spread-Cadl.cadl

```diff
@@ -34,8 +34,48 @@ model Thing {
   age: int32;
 }
 
+alias ThingAlias = {
+  @doc("name of the Thing")
+  name: string;
+  @doc("age of the Thing")
+  age: int32;
+};
+
+alias ThingAliasWithModel = {
+  @path id: string;
+  ...Thing;
+  @header top: int32;
+};
```

Comment:
Could we also have a case exactly like this but the `Thing` here is another alias? Something like
```
alias Alias1 = {
   @path id: string;
   ...Alias2;
   @header top: int32;
};
```
We should expect all the properties in Alias1 and Alias2 are parameters of the same method signature.

Reply:
Yes. This is possible scenario. I will add one. Thanks